apim/4.x/build/files/graviteeio-apim-management-ui.conf

```diff
@@ -1,8 +1,36 @@
 server {
 	listen  0.0.0.0:8084;
-	root    /opt/graviteeio/apim/management-ui;
 
-	location / {
-		index index.html;
-	}
+    add_header X-Frame-Options "SAMEORIGIN" always;
+    add_header Content-Security-Policy "frame-ancestors 'self';" always;
+    add_header X-XSS-Protection "1; mode=block";
+    add_header X-Content-Type-Options nosniff;
+    add_header X-Permitted-Cross-Domain-Policies none;
+
+    index index.html index.htm;
+    charset utf-8;
+
+    gzip on;
+    gzip_disable "msie6";
+
+    gzip_vary on;
+    gzip_proxied any;
+    gzip_comp_level 6;
+    gzip_buffers 16 8k;
+    gzip_http_version 1.1;
+    gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript application/javascript;
+
+    location / {
+        try_files $uri $uri/ =404;
+        error_page 404 /;
+        root /opt/graviteeio/apim/management-ui;
+        sub_filter '<base href="/"' '<base href="/"';
```

Comment:
Add a comment to explain how to configure the console_base_href.
See: https://github.com/gravitee-io/gravitee-api-management/blob/8c8bced397a0ce7b0244bf1d444deedc49c33eb4/gravitee-apim-console-webui/docker/config/default.conf#L31

For instance if console is served by: https://my-apim-domain/console
instead of https://console.my-apim-domain/